llmspec/llmspec.py

```diff
@@ -52,7 +53,9 @@ def get_conversation_prompt(self, messages: List[ChatMessage]) -> str:
                 message_prefix = self.assistant_token
             else:
                 message_prefix = self.system_token
-            formatted_messages.append(f"{message_prefix}{message.content}")
+            formatted_messages.append(
+                f"{message_prefix}{message.content}{self.sep_token}"
+            )
         conversation = "\n".join(formatted_messages)
```

Comment:
The messages will be joined with `/n`, does them require extra `\n`?

Reply:
Every message should end up with a sep_token.

Reply:
And it should be configurable. Let me update it.